src/content/users/ProfileView.tsx

```diff
@@ -460,9 +496,14 @@ const ProfileView: FC<Props> = ({ _id, viewType }: Props) => {
                             </StyledTag>
                           );
                         }}
-                        options={studyOptions}
+                        options={
+                          roleField === "Federal Lead" &&
+                          !studyOptions?.includes(ALL_STUDIES_OPTION)
+                            ? [ALL_STUDIES_OPTION, ...studyOptions]
+                            : studyOptions
+                        }
                         getOptionLabel={(option: string) => formattedStudyMap[option]}
-                        onChange={(_, data: string[]) => field.onChange(data)}
+                        onChange={(_, data: string[]) => handleStudyChange(field, data)}
```

Comment:
Unsure of where exactly to put this, but related to your issue where you switch from another role to "Federal Lead". After you switch from another role and click the studies dropdown, the first click doesn't actually select any new studies. After the first click the rest work as expected. Even with my fix suggestion this doesn't go away. If this is a MUI thing, then disregard.